### PR TITLE
Example to decrypt from old and encrypt in new network

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
### Fixes Issue: [LIT-2534](https://linear.app/litprotocol/issue/LIT-2534/example-to-decrypt-from-old-networks-and-encrypt-in-the-newer-ones)

# Description

We now use ID encryption which is different from Jalapeno & Serrano

Devs find it difficult to understand how to do this migration

Create an example for doing so explaining each step & differences between the two encryption schemes

Trying to update the docs for it!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Introducing new feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Review by glitch003 — APPROVED** [review]
looks good, thanks

File: docs/network/migration-guide.md [hunk 1]
```diff
@@ -43,11 +43,118 @@ We have an example script here that will perform the re-minting for you: https:/
 
 ## Performing re-encryption
 
-Re-encryption is simply, decrypting the content, then encrypting it again.  In the case of a migration from an old Lit Network to Habanero, you would connect to the old network, decrypt the user’s data, and then connect to Habanero and encrypt it again.
+Re-encryption is simply, decrypting the content, then encrypting it again. The v3 SDK of the encryption system introduces significant enhancements compared to v2 SDK. It employs a more intricate yet secure method of encryption and decryption, utilizing hashes and a comprehensive set of parameters to bolster security and integrity.
```
Comment (Adarsh-Kumar28) [inline docs/network/migration-guide.md:46]: Add about ID based encryption and that the encryption part happens completely client-side: https://developer.litprotocol.com/v3/sdk/access-control/encryption#how-does-id-encrypt-work

**Review by Adarsh-Kumar28 — CHANGES_REQUESTED** [review]
Looks good a minor comment

File: docs/network/migration-guide.md (hunk 1 above)
Comment (Aaryan-R-S) [reply docs/network/migration-guide.md:46]: Done @Adarsh-Kumar28 :)

**Review by Adarsh-Kumar28 — APPROVED** [review]
LGTM!

**Pull request merged**